testapp/build.gradle

```diff
@@ -51,6 +52,7 @@ android {
             projectId               : property("HOST_PROJECT_PROD_ID") ?: "test-host-project-id",
             subscriptionKey         : property("HOST_APP_PROD_SUBSCRIPTION_KEY") ?: "test-subs-key",
             adMobAppId              : property("PROD_ADMOB_APP_ID") ?: "",
+            appCenterSecret         : property("PROD_APPCENTER_SECRET") ?: "",
```

Comment:
I think we only need `APPCENTER_SECRET` because there is only single AppCenter to manage both staging and production apk.